open-media-match/src/OpenMediaMatch/app.py

```diff
@@ -72,6 +81,36 @@ def status():
         """
         return "I-AM-ALIVE\n"
 
+    @app.route("/", methods=["POST"])
+    def upload():
+        signaltypes = get_all_signal_types()
+        contenttypes = get_all_content_types()
+        banks = banks_index()
+        if request.method == "POST":
+            # content type from dropdown form
+            contenttype = request.form.get("media")
+            f = request.files["file"]
+            f.save(f.filename)
+            files = {
+                contenttype: open(f.filename, "rb"),
+            }
+            # returns a dictionary of {'signaltype' : 'hash'}
+            r = requests.post("http://localhost:5000/h/hash", files=files)
```

Comment:
blocking: This will only work in the development environment. Instead you can import the hashing blueprints and call them directly instead